pkg/noderesourcetopology/score.go

```diff
@@ -65,17 +65,20 @@ func (tm *TopologyMatch) Score(ctx context.Context, state *framework.CycleState,
 	nodeTopology := findNodeTopology(nodeName, tm.lister)
 
 	if nodeTopology == nil {
+		klog.V(4).InfoS("noderesourcetopology was not found for node", "node", nodeName)
```

Comment:
maybe V(5).

Reply:
I thought that since it's unusual/unhealthy behavior it should be at a higher (or low depending on how you look at this) verbosity level, but ok I changed it.